api/test/integration/listing.e2e-spec.ts

```diff
@@ -1029,4 +1025,30 @@ describe('Listing Controller Tests', () => {
       );
     });
   });
+
+  describe('mapMarkers endpoint', () => {
+    it('should find all active listings', async () => {
+      const listingData = await listingFactory(jurisdictionAId, prisma);
+      const listing = await prisma.listings.create({
+        data: listingData,
+      });
+
+      const closedListingData = await listingFactory(jurisdictionAId, prisma, {
+        status: ListingsStatusEnum.closed,
+      });
+      const closedListing = await prisma.listings.create({
+        data: closedListingData,
+      });
+
+      const res = await request(app.getHttpServer())
+        .get('/listings/mapMarkers')
+        .expect(200);
+
+      expect(res.body.length).toBeGreaterThanOrEqual(1);
+
+      const ids = res.body.map((marker) => marker.id);
+      expect(ids).toContain(listing.id);
+      expect(ids).not.toContain(closedListing.id);
```

Comment:
I know this is a release PR so it makes more sense to update the test in core, but we should also verify that the lat and long is returned

Reply:
We can add that in with your next backend change to core